cmd/kube-apiserver/app/server.go

```diff
@@ -510,13 +514,26 @@ func getRuntimeConfigValue(s *options.APIServer, apiKey string, defaultValue boo
 	return defaultValue
 }
 
-// Parses the given runtime-config and formats it into map[string]ApiGroupVersionOverride
-func parseRuntimeConfig(s *options.APIServer) (map[string]genericapiserver.APIGroupVersionOverride, error) {
+// Parses the given runtime-config and formats it into genericapiserver.APIResourceEnablement
+func parseRuntimeConfig(s *options.APIServer) (genericapiserver.APIResourceEnablement, error) {
+	v1GroupVersionString := "api/v1"
+	extensionsGroupVersionString := "extensions/v1beta1"
+	batchGroupVersionString := "batch/v1"
+	autoscalingGroupVersionString := "autoscaling/v1"
+	versionToEnablementString := map[unversioned.GroupVersion]string{
```

Comment:
> That will not work for "api/v1". Will work for other groups.

And to maintain current behavior, the map (or map plus slice), needs to be hardcoded here.  In addition, I suspect you'll end up regretting the "group/version" structure as the key.  It doesn't work very well when expanded to multiple versions (you'll probably end up with a "*" concept) where you need multi-level controls on what you're tweaking.

I'd claim that what I've got is strictly better than what existed before.  Would `.String` on the version into the map make you happy enough?